cmd/configschema/go.mod

```diff
@@ -5,12 +5,12 @@ go 1.18
 require (
 	github.com/fatih/structtag v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.67.0
+	github.com/open-telemetry/opentelemetry-collector-contrib v0.0.0-00010101000000-000000000000
```

Comment:
This should be using the version right? not the version alias used by `replace local` ?